part2/courseinfo/src/components/Course.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
```

Comment:
u wanna split each of these components into their own files?

Reply:
I will work on this